crates/ruma-client-api/src/sync/sync_events/v5.rs

```diff
@@ -245,7 +280,7 @@ pub mod request {
         /// room subscriptions. If defined and empty, will be disabled for all
         /// the rooms.
         #[serde(skip_serializing_if = "Option::is_none")]
-        pub rooms: Option<Vec<OwnedRoomId>>,
+        pub rooms: Option<Vec<ExtensionRoomConfig>>,
```

Comment:
The to-device extension doesn't use the `lists` and `rooms` settings, which makes sense since to-device events are not sent in rooms.

Reply:
Good point, I've removed the fields.